trace/sync/kafka/kafka.go

```diff
@@ -0,0 +1,124 @@
+package kafka
+
+import (
+	"context"
+	"github.com/beatlabs/patron/trace"
+
+	"github.com/Shopify/sarama"
+	"github.com/beatlabs/patron/encoding/json"
+	"github.com/beatlabs/patron/errors"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+)
+
+// Message abstraction of a Kafka message.
+type Message struct {
+	topic string
+	body  []byte
```

Comment:
should we maybe also add the key ?
see https://github.com/beatlabs/patron/pull/86 ?

Reply:
@drakos74 I have covered it in https://github.com/beatlabs/patron/pull/170